app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -514,6 +514,14 @@ void OpenXRInputSource::UpdateHaptics(ControllerDelegate &delegate)
     CHECK_XRCMD(applyHapticFeedback(mHapticAction, duration, XR_FREQUENCY_UNSPECIFIED, pulseIntensity));
 }
 
+#ifdef SPACES
+bool shouldConsiderPoseAsValid(const XrPosef& pose) {
+    // A bug in spaces leaves the locationFlags always empty. The best we can do is to check that
```

Comment:
I know that this is just temporary code to sever as a workaound for runtime related bugs, but perhaps a DEBUG message when the execution reaches this codepath would help other devs to find explanations to buggy behavior on other use cases. WDYT ?

Reply:
Makes sense, I'll add it

Reply:
After thinking about it twice I think this makes not much sense because this will be called multiple times per frame, so we'd end up flooding the log unnecessarily.